From 1b815966c199926d91309cea3a6d305a5fd8c9e9 Mon Sep 17 00:00:00 2001
From: Yongxin Liu <yongxin.liu@windriver.com>
Date: Wed, 20 Mar 2024 09:15:56 +0000
Subject: [PATCH] rdk/grr: klm/cpk/ice_txrx.c: include xdp.h

Due to kernel commit 680ee0456a57 ("net: invert the netdevice.h vs xdp.h
dependency"), include xdp.h.

Upstream-Status: Inappropriate [RDK is released in tarball only]

Signed-off-by: Yongxin Liu <yongxin.liu@windriver.com>
---
 klm/cpk/ice_txrx.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/klm/cpk/ice_txrx.c b/klm/cpk/ice_txrx.c
index 9eae76c..b783a35 100644
--- a/klm/cpk/ice_txrx.c
+++ b/klm/cpk/ice_txrx.c
@@ -6,6 +6,7 @@
 #include <linux/mm.h>
 #include <linux/netdevice.h>
 #include <linux/prefetch.h>
+#include <net/xdp.h>
 #include "ice_txrx_lib.h"
 #include "ice_lib.h"
 #include "ice.h"
-- 
2.39.3

